pkg/memo/codec_compact.go

```diff
@@ -0,0 +1,260 @@
+package memo
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+)
+
+var _ Codec = (*CodecCompact)(nil)
+
+// CodecCompact is a coder/decoder for compact encoded memo fields
```

Comment:
I think we could have more details on the usage of this one

Reply:
added more description https://github.com/zeta-chain/node/pull/2987/commits/d022be1053ebba9c11d5d3d5c51755b3705b4c03